book/src/getting-started/quickstart.md

```
@@ -0,0 +1,231 @@
+  format: console
+```
+
+**Run it:**
+
+```bash
+./logfwd --config config.yaml
+```
+
+You'll see a startup banner followed by colored output for every log line:
+
+```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)